README.md

```diff
@@ -98,14 +103,92 @@ We have a small collection of algorithms, one for every day of the month. Scroll
       left => right
     ```
 
+###  **December 6 - LCM**
+- **Problem**
+  - Return the least common multiple of two or more numbers.
+  - Use the `greatest common divisor (GCD)` formula and the fact that `lcm(x,y) = x * y / gcd(x,y)` to determine the least common multiple.
+  - **Hint:** The GCD formula might use recursion.
```

Comment:
Change this to something like "There are multiple ways to implement GCD Take a leap and try doing it with recursion."